x-pack/platform/plugins/private/upgrade_assistant/server/lib/es_deprecations_status/index.ts

```diff
@@ -56,14 +56,22 @@ export async function getESUpgradeStatus(
       return status !== 'green';
     }) as EnrichedDeprecationInfo[];
 
-    return [...enrichedHealthIndicators, ...toggledMigrationsDeprecations];
+    return {
+      enrichedHealthIndicators,
+      migrationsDeprecations: toggledMigrationsDeprecations,
```

Comment:
naming is hard. `migrationsDeprecations` inspired by `toggledMigrationsDeprecations` by dropping the `toggled` part.
Open to changing it if the var name isn't appropriate.